app/views/posts/index.html.erb

```diff
@@ -1 +1,38 @@
-<h1>Here is a list of all posts</h1>
\ No newline at end of file
+<br/>
+<div class="container-sm">
+  <div class="card mb-3">
+    <div class="row g-0">
+      <div class="col-md-4" style="width: 100px;">
+        <img src="<%= @user.photo %>" class="img-fluid rounded" alt="User Photo">
+      </div>
+      <div class="col-md-8">
+        <div class="card-body">
+          <h5 class="card-title"><%= @user.name %></h5>
+          <p class="card-text"><small class="text-muted">Number of posts: <%= @user.post_counter %></small></p>
+        </div>
+      </div>
+    </div>
+  </div>
+  <% @posts.each do |post| %>
+    <div class="card border-success mb-3">
+      <div class="card-body text-success">
+        <a href = '<%= user_post_path(@user, post) %>'>
+          <h5 class="card-title">Post #<%= @posts.index(post) + 1 %></h5>
```

Comment:
- [ ] Great job working on the views but I think you have an issue here since when I press on a post instead of leading me to the show page it shows this error so please try to solve the issue here so I can see the show page 🎉⭐✨


![image](https://github.com/noelfoka/Blog_app/assets/58553711/9a90aea5-c59b-41d4-a4c6-e85d15f62d99)



- I should see something like this ⬇⬇
![image](https://github.com/noelfoka/Blog_app/assets/58553711/956ae469-7ae6-42b1-b44e-558144e95c0e)

Reply:
This issue has been resolved